src/SFML/Graphics/RenderTextureImplFBO.cpp

```diff
@@ -116,17 +116,16 @@ bool RenderTextureImplFBO::isAvailable()
 ////////////////////////////////////////////////////////////
 unsigned int RenderTextureImplFBO::getMaximumAntialiasingLevel()
 {
-    const TransientContextLock lock;
-
-    GLint samples = 0; // NOLINT(misc-const-correctness)
-
 #ifndef SFML_OPENGL_ES
 
+    const TransientContextLock lock;
+    GLint                      samples = 0;
     glCheck(glGetIntegerv(GLEXT_GL_MAX_SAMPLES, &samples));
+    return static_cast<unsigned int>(samples);
 
 #endif
 
-    return static_cast<unsigned int>(samples);
+    return 0;
```

Comment:
Surprised that Clang isn't complaining about unreachable code.

I wonder if it weren't more readable, by having a `#ifdef SFML_OPENGL_ES` section at the top, that just returns 0.

Reply:
I'd prefer that too